### ISSUE TITLE
kong fails to start due to missing serf

### ISSUE THREAD

**Issue by gsgleason** [Mashape/kong#1953]
### Summary
I deployed a brand new Centos7 machine and fully updated it.

I installed Kong per installation guide (https://getkong.org/install/centos/), using this rpm: https://github.com/Mashape/kong/releases/download/0.9.7/kong-0.9.7.el7.noarch.rpm

I copied /etc/kong/kong.conf.defaut to /etc/kong/kong.conf and updated my database information.

However, kong fails to start:

Error:
/usr/local/share/lua/5.1/kong/cmd/start.lua:37: /usr/local/share/lua/5.1/kong/cmd/start.lua:22: could not find 'serf' executable (is 'serf_path' correctly set?)

Two solutions worked:

1: add to kong.conf: serf_path = /usr/local/bin/serf
2: add /usr/local/bin to root path

Simply put, following the installation instructions on a default CentOS 7 install will result in kong not starting due to serf not being found.

### Steps To Reproduce

1.
2.
3.
4.

### Additional Details & Logs

- Kong version (`$ kong version`)
- Kong debug-level startup logs (`$ kong start --vv`)
- Kong error logs (`<KONG_PREFIX>/logs/error.log`)
- Kong configuration (registered APIs/Plugins & configuration file)
- Operating System

**Comment by koolay** [comment]
not work.

step 1: add to kong.yml: serf_path: /usr/local/bin/serf

**Comment by Tieske** [comment]
@koolay what version do you use? `kong.yml` has been deprecated for quite some time now.

**Comment by koolay** [comment]
@Tieske 

0.9.5

**Comment by Tieske** [comment]
fixed see #1997
